tests/IdentifyServiceProviderTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Unicodeveloper\Identify\Test;
+
+use Mockery\MockInterface;
+use PHPUnit_Framework_TestCase;
+use Unicodeveloper\Identify\IdentifyServiceProvider;
+
+class IdentifyServiceProviderTest extends PHPUnit_Framework_TestCase
+{
+
+    /**
+     * @var MockInterface
+     */
+    protected $mockApp;
+
+    /**
+     * @var IdentifyServiceProvider
+     */
+    protected $provider;
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->mockApp = \Mockery::mock();
+        $this->provider = new IdentifyServiceProvider($this->mockApp);
+    }
+
+    /**
+     * Test that it provides the correct service name
+     */
```

Comment:
Lol, I don't think this comment is doing justice to what you are testing in this method.